### PR TITLE
supplemental: enforce coverage ratio for gui tests to the current values #11514

### PR DESCRIPTION
**Pull request by pbludov**
supplemental PR for #11514 

Coverage percentage should be explicitly specified in the `pom.xml` file to prevent coverage decrease.

### CODE REVIEW

**Review by romani — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -875,15 +875,21 @@
                       com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocMethodCheck.RegularClass
                     </exclude>
                     <!-- Swing related classes -->
-                    <exclude>com.puppycrawl.tools.checkstyle.gui.BaseCellEditor*</exclude>
+                    <exclude>com.puppycrawl.tools.checkstyle.gui.BaseCellEditor</exclude>
```
Comment (nick-mancuso) [inline pom.xml:878]: Can we remove all classes in `gui`  from excludes in this PR?

**Review by nick-mancuso — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (rnveach) [reply pom.xml:878]: I am confused why these excludes are changing to only remove the `*` on the end? I believe these stars to are include any subclasses, so we are changing it to account for the subclasses but ignore the primary classes? Shouldn't these excludes be fully removed and we will keep the individual coverage on the bottom?

Comment (pbludov) [reply pom.xml:878]: That is exactly what I did. `TreeTable` and `TreeTable.TreeTableCellEditor` have their own coverage config now.
Same for `TreeTableModelAdapter` and `TreeTableModelAdapter.UpdatingTreeModelListener`.

All other classes do not have inner classes. For them, `*` has no effect.

Comment (nick-mancuso) [reply pom.xml:878]: @pbludov my mistake, I thought that once we defined coverage rules elsewhere, that we did not need thees excludes anymore. resolved.

**Review by nick-mancuso — APPROVED** [review]

**Review by rnveach — APPROVED** [review]
I don't see why we are removing the stars. Even if there are no subclasses, for them, they can remain until the entire exclude is removed.

**Comment by romani** [comment]
If subclasses appears, jacoco will fail as far as I remember

**Pull request merged**